packages/appsync-modelgen-plugin/src/visitors/appsync-visitor.ts

```diff
@@ -102,6 +102,13 @@ export interface RawAppSyncModelConfig extends RawConfig {
    * @descriptions optional boolean which adds the read-only timestamp fields or not
    */
   isTimestampFieldsAdded?: boolean;
+
+  /**
+   * @name directives
+   * @type boolean
+   * @descriptions optional boolean, if true emits the provider value of @auth directives
+   */
+   emitAuthProvider?: boolean;
```

Comment:
The same as the previous index PR. This config only affects swift and it is better to move the type definition and config parsing into `appsync-swift-visitor`

Reply:
Will do in next revision